packages/perseus/src/validation.types.ts

```diff
@@ -24,6 +24,8 @@ export type PerseusCategorizerRubric = {
     // The correct answers where index relates to the items and value relates
     // to the category.  e.g. [0, 1, 0, 1, 2]
     values: ReadonlyArray<number>;
+    // Translatable text; a list of items to categorize. e.g. ["banana", "yellow", "apple", "purple", "shirt"]
+    items: ReadonlyArray<string>;
```

Comment:
At first, I was just using `values` for validation, but since `items` is already sent to the scoring function, I just added it back to `rubric` so `values` can be stripped away during the rubric stripping step and validation still works. Totally up for changing this however though.

Reply:
We chatted on Slack, but we agreed to model the Rubric like this: 

```ts
export type PerseusCategorizerRubric = {
     // The correct answers where index relates to the items and value relates
     // to the category.  e.g. [0, 1, 0, 1, 2]
     values: ReadonlyArray<number>;
} PerseusCategorizerValidationData; 
```

We also agreed to rename the term `Rubric` to `ScoringData` in the Server-Side Scoring area in a future task. :)

Reply:
The above comment I made was from an older version. I updated Rubric to be the following:
```
export type PerseusCategorizerScoringData = {
    // The correct answers where index relates to the items and value relates
    // to the category.  e.g. [0, 1, 0, 1, 2]
    values: ReadonlyArray<number>;
} & PerseusCategorizerValidationData;
```

I assumed we could change Rubric as we go and the ticket would be for updating any we missed. It helps my mental model to have the names updated while I'm working, but I can totally leave the name change to the very end if it's preferred to do them all at once?

Reply:
I started renaming things to `ScoringData` in one of my PRs. Then as I thought about it, I thought it might be more confusing to have a partial migration state in `main`. I was going to leave things as "Rubric" until we're done this pass of validation splitting and then we could do a single pass over everything to update our terms from Rubric to ScoringData.  But I don't think it's a huge deal either way. :)